src/holodeck/packagemanager.py

```diff
@@ -315,13 +315,17 @@ def get_scenario(scenario_name):
 
 
 def get_binary_path_for_package(package_name):
-    """Gets the path to the binary of a specific package
+    """Gets the path to the binary of a specific package. Raises
```

Comment:
remove word raises

Reply:
Did you mean to document the possible exceptions that can be raised?

Reply:
Yes but then I gave up